templates/components/event_hero.html

```diff
@@ -0,0 +1,28 @@
+{% load wagtailimages_tags %}
+<div class="event-hero">
+    <div class="event-hero__breadcrumb-mobile">
+        {% include 'includes/breadcrumb.html' %}
+    </div>
+    <div class="event-hero__info-wrapper">
+        <div class="event-hero__breadcrumb-desktop">
+            {% include 'includes/breadcrumb.html' %}
```

Comment:
We can't have two sets of breadcrumbs in the HTML - this would show up when you viewed the page without CSS.

Reply:
Ah good point. I chatted with Nick and Lauren this morning - they're happy to move the image below the text at mobile which will make the layout more straightforward, and we can lose the repeated markup.